tests/ClosestCity/ClosestCity.cpp

```diff
@@ -15,119 +15,189 @@
 #include <algorithm>
 #include <cmath>
 #include <gtest/gtest.h>
-bool findCity(std::vector <City> vec, const std::string &city_name){
-    for(auto it = vec.begin();it!=vec.end();++it){
-        if(it->getName()==city_name) return true;
+bool findCity(std::vector<City> vec, const std::string &city_name) {
+    for (auto it = vec.begin(); it != vec.end(); ++it) {
+        if (it->getName() == city_name)
+            return true;
     }
     return false;
 }
 
 TEST(ClosestCity, ClassFunctionsTests) {
 
-    //testing removeCitiesInVectorThatAreSmallerThanMe
-    //For Warsaw we should have only one city - Warsaw
-    //For others we should get more cities
-    //TODO:vector pobierany z pliku moc 10 cities databaseReaderem.
+    // testing removeCitiesInVectorThatAreSmallerThanMe
+    // For Warsaw we should have only one city - Warsaw
+    // For others we should get more cities
+    // TODO:vector pobierany z pliku moc 10 cities databaseReaderem.
     std::string path = "../../../database/Poland-Cities010-Top10-mock.txt";
-    std::vector <City> vec = DatabaseReader::readCitiesFromFile(path);
-    
-    std::string CityNames[]={"Warsaw","Lodz","Szczecin", "Katowice", "Poznan","Krakow"};
-    ClosestCity::removeCitiesInVectorThatAreSmallerThanMe(vec,CityNames[0]);
-    ASSERT_EQ(1, vec.size()) << "We should erase cities smaller than Warsaw and therefore have just one city in vector but we have: " << vec.size() << " of them";
-    ASSERT_TRUE(findCity(vec,CityNames[0])) << "We should have one " << CityNames[0] << " but we found" << findCity(vec,CityNames[0]);
-    ASSERT_FALSE(findCity(vec,CityNames[1])) << "We should have none " << CityNames[1] << " but it is still in the vector so the function removeCitiesInVectorThatAreSmallerThanMe is working wrong";
+    std::vector<City> vec = DatabaseReader::readCitiesFromFile(path);
```

Comment:
`DatabaseReader::readCitiesFromFile(path);` returns `std::vector<City>` so why do we need to write is explicitly?
That's just an open question. Perhaps it is fine to write `std::vector<City>` insted of auto.

Reply:
No you are right, I should use auto. There is no reason for keeping explicit definition.